device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "sesto-1",
+      "address": "0x1000",
+      "lib_name": "libsai_phy_sesto-1.so",
+      "firmware_path": "/tmp/phy-sesto-1.bin",
+      "config_file": "/usr/share/sonic/hwsku/sesto-1.json",
+      "sai_init_config_file": "/usr/share/sonic/hwsku/sesto-1.bcm",
+      "phy_access": "mdio",
+      "bus_id": 0
+    }
+  ],
+  "interfaces": [
+    {
+      "name": "Ethernet48",
+      "index": 49,
+      "phy_id" : 1,
+      "system_lanes": [200,201,202,203],
+      "line_lanes": [204,205]
+    },
+    {
+      "name": "Ethernet49",
+      "index": 50,
+      "phy_id" : 1,
+      "system_lanes": [206,207,208,209],
+      "line_lanes": [210,211]
+    },
+    {
+      "name": "Ethernet50",
+      "index": 51,
+      "phy_id" : 1,
+      "system_lanes": [212,213,214,215],
```

Comment:
i think we are missing the system_lanes to asic lane mapping here. otherwise, how can we know which asic lane mapps to which gear box system side lanes?

Reply:
The original mapping is preserved in the associated port_config.ini file. Please note that there is a DPB proposal such that the new platform.json will deprecate the old port_config.ini file at which time, the gearbox_config.json will need to be augmented.